Assets/Scripts/UI/ObjectiveList/ObjectiveScriptableObjects/Kitchen1/dentures.asset

```diff
@@ -12,4 +12,4 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 50d658d4821ea4345a962a7a31850b5a, type: 3}
   m_Name: dentures
   m_EditorClassIdentifier: 
-  line: Steal dentures for Wilbur
+  line: Steal dentures for Wilbur from the Kitchen
```

Comment:
Aren't we acquiring Wilbur's dentures? Shouldn't this be something like "Acquire Wilbur's dentures in the kitchen"

Reply:
ah true yea, I'll change it to that!